Patches/Forsakens/Forsakens_CE_Patch_PawnKinds_Forsaken.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Forsakens</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<!-- ========== Reduce meals and medicine carried by all pawns ========== -->
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/PawnKindDef[
+						@Name="ForsakenBase" or
+						defName="ForsakenSniper" or
+						defName="ForsakenSlasher"
+					]/invNutrition</xpath>
+					<value>
+						<invNutrition>1</invNutrition>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/PawnKindDef[
+					defName="ForsakenSlasher" or
+					defName="ForsakenGunner" or
+					defName="KingOfTheLostOnes"
+				]/inventoryOptions/subOptionsChooseOne</xpath>
+					<value>
+						<subOptionsChooseOne>
+							<li>
+								<thingDef>Silver</thingDef>
+								<countRange>
+									<min>180</min>
+									<max>400</max>
+								</countRange>
+							</li>
+							<li>
+								<thingDef>MedicineIndustrial</thingDef>
+								<countRange>
+									<min>1</min>
```

Comment:
Can we set min to 0? It's probably going to end up being a lot of medicine anyway, and frankly I don't know if pawns from other factions will use it when healing themselves (and most will probably never get the chance, I'd wager).

Reply:
Yup, can do.

Fixed in dc071573